sdk/textanalytics/Azure.AI.TextAnalytics/src/DetectedLanguage.cs

```diff
@@ -13,16 +13,15 @@ namespace Azure.AI.TextAnalytics
     public readonly struct DetectedLanguage
     {
         internal DetectedLanguage(DetectedLanguageInternal language, IList<TextAnalyticsWarning> warnings)
-            : this(language.Name, language.Iso6391Name, language.ConfidenceScore, language.Script, warnings)
```

Comment:
I see various `DetectedLanguage` properties were removed, so is this type even needed? Or is it still used elsewhere that's hard to tell only from the context of this PR?

Reply:
It's still used elsewhere. The `DetectedLanguage` properties that were removed were part of a preview feature called automatic language detection, which allowed all the other skills (such as sentiment analysis or summarization) to automatically detect the language of the input before actually performing the corresponding skill on it. There is still a separate skill available that only does language detection, and this `DetectedLanguage` type is used for that.